public/js/playerGame.js

```diff
@@ -101,3 +101,11 @@ socket.on('GameOver', function(){
     document.getElementById('message').innerHTML = "GAME OVER";
 });
 
+
+/*
+ * Web socket to refresh page (usually in case when 
+ * a user with same name is already in the game)
+ */
+socket.on('PageRefresh', function(){
+    location.reload();
```

Comment:
Instead of reloading page, show alert to the user that the user with same name is already in the lobby!

Reply:
That can be done in future of-course!